tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -71,6 +74,10 @@ public TiTableInfo(
     this.charset = charset;
     this.collate = collate;
     this.columns = ImmutableList.copyOf(requireNonNull(columns, "columns is null"));
+    this.columnsMap = new HashMap<>();
+    for (TiColumnInfo col : this.columns) {
+      this.columnsMap.put(col.getName(), col);
```

Comment:
question: is this map case sensitive?

Reply:
nope. It is case insensitive.

Reply:
then how to make sure we choose the correct column?

Reply:
You can check `col.getName()`. The `name` of `TiColumnInfo` is always little case. In `getColumn()` of `TiTableInfo`, we always use lower case to get correct `TiColumnInfo`. 
Therefore, we can make sure we always choose the correct column.